### ISSUE TITLE
ssl cert for lists.codespeak.net broken

### ISSUE THREAD

**Issue by hpk42** [codespeaknet/sysadmin#8]
![lists-cert](https://user-images.githubusercontent.com/73579/36719668-fc7b9f1a-1ba5-11e8-9b35-4f699282d1f7.png)

seems our lists.codespeak.net cert is not properly served.  Pretty sure it used to work.

**Comment by gfa** [comment]
Solved by d70ba88dbe408e54ac4f1389119985c9a8213a87

**Comment by gfa** [comment]
For posterity!

```
diff --git a/nginx/sites-available/lists b/nginx/sites-available/lists
index b1587db..2de3f3b 100644
--- a/nginx/sites-available/lists
+++ b/nginx/sites-available/lists
@@ -1,5 +1,6 @@
 server {
     listen 80;
+    listen [::]:80;

     server_name lists.codespeak.net;

@@ -14,6 +15,7 @@ server {

 server {
     listen 443 ssl;
+    listen [::]:443 ssl;
     server_name lists.codespeak.net;

     ssl_certificate /etc/dehydrated/certs/lists.codespeak.net/fullchain.pem;
```

nginx wasn't listening on ipv6 for that vhost, sending the traffic to the default vhost
